### ISSUE TITLE
Customize colors in Terms panel ?

### ISSUE THREAD

**Issue by aeddaifi** [lucidworks/banana#176]
Hello,
I'd like to know if is it possible to customize colors in Terms panel ?
For example, I have a panel showing the number of calls by Country, I would like to define a different color for each country.
Defining colors is made possible for the Query panel, but I could not find anything similar for Terms panel.
Thanks in advance !

**Comment by aadel** [comment]
Hi @aeddaifi there's an option at the bottom of the Terms panel settings to customize colors "Default Colors for Chart". Colors, in hexadecimal, are assigned respectively to bars/pie segments, i.e. the first color goes to the 1st top, second goes to the 2nd top and so on.

**Comment by rdubreui** [comment]
Yes, but if your 1st top and 2nd top are not the same after 10 minutes and that your term changes its colours many time and an object marked blue can be green…in my opinion, it is very counterproductive.
Can we associate a colour to a value ?
